lib/12_rpn_calculator.rb

```diff
@@ -1,3 +1,66 @@
 class RPNCalculator
-  # TODO: your code goes here!
+  attr_accessor :arr
+
+  def initialize
+    @arr = []
+  end
+
+  def push(num)
+    arr << num
+  end
+
+  def plus
+    check_if_empty
+    added = arr.pop(2).inject(0, :+)
```

Comment:
I think `reduce(:+)` will accomplish the same as `inject(0, :+)`, not as verbosely. `reduce` is what I use always for summing.